### PR TITLE
Allow packager to specify source repo for b2d CLI/ISO

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
- Moved bucket/src strings below
- Separated versions for b2d ISO and CLI.
- Added separate src repo options for b2d ISO and CLI

Signed-off-by: Ahmet Alp Balkan <ahmetalpbalkan@gmail.com>
cc:@tianon

### CODE REVIEW

File: bundle.sh [hunk 1]
```diff
@@ -4,13 +4,17 @@ set -e
...
+boot2dockerIso=1.5.0
+boot2dockerCli=1.5.0
 docker=1.6.0-rc1
-dockerBucket=test.docker.com
 vbox=4.3.26
 vboxRev=98988
 msysGit=1.9.5-preview20150319
 
+boot2dockerIsoSrc=boot2docker
+boot2dockerCliSrc=boot2docker
```
Comment (tianon) [inline bundle.sh]: Do you think these should be the full `boot2docker/boot2docker` in case someone renames their fork?

Comment (ahmetalpbalkan) [inline bundle.sh]: @tianon would be very rare. not worth the confusion IMO.

Comment (tianon) [inline bundle.sh]: Fair enough -- can always change it if it's a problem. :+1:

**Comment by tianon** [comment]
LGTM

**Pull request merged**